### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by darkdef**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Fixed issues  |

### CODE REVIEW

File: src/MessageSource.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+declare(strict_types=1);
+
+namespace Yiisoft\Translator\Message\Php;
+
+use Yiisoft\Translator\MessageReaderInterface;
+use Yiisoft\Translator\MessageWriterInterface;
+
+final class MessageSource implements MessageReaderInterface, MessageWriterInterface
+{
+    private $path;
```
Comment (samdark) [inline src/MessageSource.php:12]:
```suggestion
    private string $path;
```

File: src/MessageSource.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    private $messages;
```
Comment (samdark) [inline src/MessageSource.php:13]:
```suggestion
    private array $messages = [];
```

File: src/MessageSource.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        } else {
+            $content .= $this->arrayToCode($messages);
+            $content .= ';';
+        }
+        $content .= "\n";
+
+        return $content;
+    }
+
+    private function arrayToCode($array, $level = 0)
```
Comment (samdark) [inline src/MessageSource.php:82]:
```suggestion
    private function arrayToCode(array $array, int $level = 0): string
```

**Review by samdark — CHANGES_REQUESTED** [review]

**Review by samdark — APPROVED** [review]

**Comment by darkdef** [comment]
@samdark  Thanks :+1:

**Comment by darkdef** [comment]
@samdark  Thanks :+1:

**Pull request merged**